man/systemd-nspawn.xml

```diff
@@ -306,6 +306,9 @@
         dm-verity if the integrity data is passed using the <option>--root-hash=</option> and
         <option>--verity-data=</option> (and optionally <option>--root-hash-sig=</option>) options.</para>
 
+        <para>If it is a dm-verity image, and the exact same volume with the same root hash was already
+        in use by a different service, it will be re-used to avoid another costly mapper operation.</para>
```

Comment:
this sounds like an implemenation detail, that might not be something we want to document (because we want to change it one day, and it's mostly noise for most readers)

Reply:
removed